app/_includes/how-tos/steps/konnect-identity-generate-token.md

```diff
@@ -0,0 +1,19 @@
+## Generate a token for the client
+
+The Gateway Service requires an access token from the client to access the Service. Generate a token for the client by making a call to the issuer URL:
+
+<!--vale off-->
+```sh
+curl -X POST "$ISSUER_URL/oauth/token" \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -d "grant_type=client_credentials" \
+  -d "client_id=$CLIENT_ID" \
+  -d "client_secret=$CLIENT_SECRET" \
+  -d "scope=Scope"
```

Comment:
```suggestion
  -d "scope=my-scope"
```
`Scope` just gave me `{"error":"invalid_scope"}`, but passing the name of the scope worked.